packages/oats-runtime/src/make.ts

```diff
@@ -352,24 +352,56 @@ export function makeOneOf(...options: any[]) {
             return error('multiple preferred options match');
           }
           preferredSuccess = mapped;
-        } else if (success) {
-          return error('multiple options match');
+        } else if (0 < success.length) {
```

Comment:
its really stupid nitpick but this reads easier in english
```suggestion
        } else if (success.length > 0) {
```

Reply:
yeah my brain froze for a second there when I saw it 😂 somehow feels like wrong syntax

Reply:
Fine, done 😂